experiments/react-native-uifabric/src/hooks/RadioGroup.hooks.ts

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+//import { IKeyboardEvent } from '@office-iss/react-native-win32';
+
+export type onChangeCallback = (key: string) => void;
+
+// This hook updates the selected button and calls the client's onChange callback with the new button
+export function useAsRadioGroupSelection(defaultSelectedKey: string, userCallback?: onChangeCallback) {
+  const [selectedKey, setSelectedKey] = React.useState(defaultSelectedKey);
+
+  const onChange = React.useCallback(
+    (key: string) => {
+      setSelectedKey(key);
+      userCallback && userCallback(key);
+    },
+    [setSelectedKey, userCallback]
```

Comment:
not that it makes a huge difference, but memoize on selectedKey instead

Reply:
Replacing userCallback with selectedKey?

Reply:
no, I'm talking about `setSelectedKey`, memoizing on that feels more correct.